tests/upgrade/test_crossgrade.sh

```diff
@@ -327,19 +345,19 @@ kubectl logs -f -n "${TEST_NAMESPACE}" -c echosrv "${cli_pod_name}" &> "${POD_FO
 local_log_str=$(grep "Code 200" "${LOCAL_FORTIO_LOG}")
 pod_log_str=$(grep "Code 200"  "${POD_FORTIO_LOG}")
 
-if [[ ${local_log_str} == "" ]]; then
+if [[ ${local_log_str} != *"Code 200"* ]];then
     echo "No Code 200 found in external traffic log"
     failed=true
-elif [[ ${local_log_str} != *"(100.0"* ]]; then
-    printf "\\n\\nErrors found in external traffic log:\\n\\n"
+elif ! percent200sAbove "${local_log_str}"; then
+    printf "\\n\\nToo many errors found in external traffic log:\\n\\n"
     cat ${LOCAL_FORTIO_LOG}
 fi
 
-if [[ ${pod_log_str} == "" ]]; then
+if [[ ${pod_log_str}  != *"Code 200"* ]]; then
     echo "No Code 200 found in internal traffic log"
     failed=true
-elif [[ ${pod_log_str} != *"(100.0"* ]]; then
-    printf "\\n\\nErrors found in internal traffic log:\\n\\n"
+elif ! percent200sAbove "${local_log_str}"; then
+    printf "\\n\\nToo many errors found in internal traffic log:\\n\\n"
     cat ${POD_FORTIO_LOG}
     exit 1
```

Comment:
and here for consistency (and this exits in one place only)

Reply:
done.